.github/workflows/lint.yml

```
@@ -0,0 +1,24 @@
+name: Lint
+
+on: push
```

```suggestion
on: [push, pull_request]
```
also trigger lint on pull requests